src/native/fs/fs_napi.cpp

```diff
@@ -1406,8 +1408,22 @@ struct GetPwName : public FSWorker
     }
     virtual void Work()
     {
-        _getpwnam_res = getpwnam(_user.c_str());
-        if (_getpwnam_res == NULL) SetSyscallError();
+        long bufsize = sysconf(_SC_GETPW_R_SIZE_MAX);
```

Comment:
instead of calling sysconf() every time we call getpwnam_r I would prefer to call it just once on fs_napi init and save in a global getpw_r_size_max.

Reply:
done